### PR TITLE
Move message from Constante in new Class Mesaje

### PR DESCRIPTION
**Pull request by alinalory**

### CODE REVIEW

File: src/com/company/Constante.java [hunk 1]
```diff
@@ -1,28 +1,11 @@
 package com.company;
 
 import java.security.PublicKey;
```
Comment (deepmikoto) [inline src/com/company/Constante.java]: Please remove this "import", it is not used

File: src/com/company/Meniu.java [hunk 2]
```diff
@@ -70,57 +70,57 @@ public  void afiseazaMeniuNouaOperatie(){
 
     }
     public void adunare(){
```
Comment (deepmikoto) [inline src/com/company/Meniu.java]: Please add an empty line between each function from every class, thank you! :smile:

File: src/com/company/OperatiiLogice.java [hunk 3]
```diff
@@ -30,27 +30,27 @@ public static void listareLista(int[] array) {
 
     public void ordonareCrescator() {
         CitireTastatura citire = new CitireTastatura();
-        System.out.println(Constante.LISTA_NUMERE);
+        System.out.println(Mesaje.lista_numere);
         int[] numere = new int[citire.citireNumar()];
         System.out.println(Constante.MARIME_LISTA + numere.length + Constante.NUMERE);
```
Comment (deepmikoto) [inline src/com/company/OperatiiLogice.java]: Please move MARIME_LISTA and NUMERE to Mesaje class

File: src/com/company/OperatiiLogice.java [hunk 4]
```diff
@@ -30,27 +30,27 @@ public static void listareLista(int[] array) {
...
 
         listareLista((sortArrayAsc(numere)));
         System.out.println("\n");
     }
     public void ordonareDescrescator() {
         CitireTastatura citire = new CitireTastatura();
-        System.out.println(Constante.LISTA_NUMERE);
+        System.out.println(Mesaje.lista_numere);
         int[] numere = new int[citire.citireNumar()];
         System.out.println(Constante.MARIME_LISTA + numere.length + Constante.NUMERE);
```
Comment (deepmikoto) [inline src/com/company/OperatiiLogice.java]: Same here, please move to Mesaje class, tahnk you!

Comment (alinalory) [reply src/com/company/OperatiiLogice.java]: Done.

File: src/com/company/OperatiiLogice.java (hunk 3 above)
Comment (alinalory) [reply src/com/company/OperatiiLogice.java]: The constants MARIME_LISTA and NUMERE are now in class Mesaje

File: src/com/company/Meniu.java (hunk 2 above)
Comment (alinalory) [reply src/com/company/Meniu.java]: added the empty line between each function.

File: src/com/company/Constante.java (hunk 1 above)
Comment (alinalory) [reply src/com/company/Constante.java]: Removed the import

**Pull request merged**